apps/nextjs-website/src/components/organisms/Auth/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,212 @@
+                  error={isPasswordDirty && !isPasswordValid}
+                  endAdornment={
+                    <InputAdornment position='end'>
+                      <IconButton
+                        aria-label='toggle password visibility'
+                        onClick={handleClickShowPassword}
+                        onMouseDown={handleMouseDownPassword}
+                        edge='end'
+                      >
+                        {showPassword ? <VisibilityOff /> : <Visibility />}
+                      </IconButton>
+                    </InputAdornment>
+                  }
+                  value={password}
+                  label={shared.password}
+                  inputProps={{
+                    sx: {
+                      padding: '8.5px 14px',
+                    },
+                  }}
+                />
+                <FormHelperText error={isPasswordDirty && !isPasswordValid}>
+                  {signUp.passwordPolicy}
+                </FormHelperText>
+              </FormControl>
+            </Stack>
+            <Stack spacing={2} mb={2}>
+              <FormControl variant='outlined'>
+                <InputLabel
+                  htmlFor='confirm-password-input'
+                  sx={{ top: '-8px' }}
+                >
+                  {shared.confirmPassword}
+                </InputLabel>
+                <OutlinedInput
+                  id='confirm-password-input'
+                  type={showPassword ? 'text' : 'password'}
+                  required
+                  onChange={({ target: { value } }) =>
+                    setPasswordConfirm(value)
+                  }
+                  endAdornment={
+                    <InputAdornment position='end'>
+                      <IconButton
+                        aria-label='toggle confirm password visibility'
+                        onClick={handleClickShowPassword}
+                        onMouseDown={handleMouseDownConfirmPassword}
+                        edge='end'
+                      >
+                        {showPassword ? <VisibilityOff /> : <Visibility />}
+                      </IconButton>
+                    </InputAdornment>
+                  }
+                  value={password_confirm}
+                  label={shared.confirmPassword}
+                  error={isPasswordDirty && password !== password_confirm}
+                  inputProps={{
+                    sx: {
+                      padding: '8.5px 14px',
+                    },
+                  }}
+                />
+              </FormControl>
+            </Stack>
+            <Stack spacing={4} pt={4} pb={2}>
+              <Stack direction='row' justifyContent='center'>
+                <Button
+                  variant='contained'
+                  onClick={onChangePassword}
+                  disabled={!isFormValid}
+                >
+                  {resetPassword.send}
+                </Button>
```

Comment:
I think user may have a better feedback if button became disable after click while waiting for a succesful redirect or to show error.
[Screencast from 16-11-2023 15:33:59.webm](https://github.com/pagopa/developer-portal/assets/24475319/dae2bc24-2b04-47f6-8e2d-bbe592e65934)